src/compiler.ts

```diff
@@ -8439,17 +8446,23 @@ export class Compiler extends DiagnosticEmitter {
         }
         this.currentType = localType;
 
-        if (target.parent != flow.parentFunction) {
-          // TODO: closures
-          this.error(
-            DiagnosticCode.Not_implemented_0,
-            expression.range,
-            "Closures"
-          );
-          return module.unreachable();
-        }
+        //if (target.parent != flow.parentFunction) {
+          //// TODO: closures
+          //this.error(
+            //DiagnosticCode.Not_implemented_0,
+            //expression.range,
+            //"Closures"
+          //);
+          //return module.unreachable();
+        //}
         return module.local_get(localIndex, localType.toNativeType());
       }
+      case ElementKind.CLOSED_LOCAL: {
+        var closedLocal = <ClosedLocal>target;
+        console.log("Accessing a closed local " + closedLocal.local.name + " at depth " + closedLocal.functionDepth + " and offset " + closedLocal.local.envOffset);
+
+        return module.i32(1);
```

Comment:
Whenever we try to access a local from an env, we'll just return a hardcoded 1 for now. Allowing the compilation to complete will help us validate that the outputted binary isn't invalid before we can replace this with an actual access